src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -170,18 +179,30 @@ public Boolean replaceKibanaIndex(final ActionRequest request, final String acti
 
         }
 
+        return CONTINUE_EVALUATION_REPLACE_RESULT;
+    }
+
+    private CreateIndexRequest checkAndCreateRequestIfAbsent(final String name) {
+        final IndexAbstraction indexAbstraction = clusterService.state().getMetadata().getIndicesLookup().get(name);
+        if (indexAbstraction == null) {
+            return new CreateIndexRequest(name + "_1")
```

Comment:
Can any suffix like "security" be used? My understanding is that migrated index will the number suffix added.

Reply:
Can the suffix logic be consolidated into a separate function to avoid missing any logic in the future. I see "_1" used in the subsequent create index logic too.

Reply:
The suffix is consistent with the suffix used by Kibana. Kibana creates `.kibana_1` index with `.kibana` alias. I'd prefer to follow the same convention.

Reply:
I was wondering if it safe to interfere with what Kibana does. My understanding is that we just shouldn't have an index terminating with a long number. As long as that is met, Kibana index migration would work correctly.

Reply:
We do not interfere with Kibana, we simply follow the same convention as Kibana.